src/lib/zendesk/zendesk.client.ts

```diff
@@ -104,6 +104,7 @@ Click this link to go to the live version of your service: ${worldpayGoLiveUrl}
 Once you’re in your account, you should confirm:
 
 * which card types you want to enable in 'Settings' (3D Secure is enabled by default)
+* wallet payment of Apple Pay is enabled by default. You can disable it in ‘Settings’.
```

Comment:
I’d just say “Apple Pay is enabled by default…”. Not sure if there should be a period at the end of the bullet point since the other bullets do not have one. Check with a content designer, maybe?